src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticIds.cs

```diff
@@ -42,5 +42,6 @@ internal static class RoslynDiagnosticIds
         public const string AnnotatePublicApiRuleId = "RS0036";
         public const string ShouldAnnotateApiFilesRuleId = "RS0037";
         public const string PreferNullLiteralRuleId = "RS0038";
+        public const string RelaxTestNamingSuppressionRuleId = "RS0039";
```

Comment:
Should we use a different prefix for suppression IDs?

Reply:
I couldn't think of a good reason to do so, or a good prefix.